tests/scripts/thread-cert/Cert_6_6_01_KeyIncrement.py

```diff
@@ -72,6 +76,54 @@ def test(self):
         for addr in addrs:
             self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ED = pv.vars['ED']
+        _leader_pkts = pkts.filter_wpan_src64(LEADER)
+        _ed_pkts = pkts.filter_wpan_src64(ED)
+
+        # Step 1: The DUT must start the network using
+        # thrKeySequenceCounter = 0
+        _leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: p.wpan.aux_sec.key_source == 0)
+
+        # Step 2: Verify that the topology described above is created.
+        # MLE Auxiliary security header shall contain Key Source = 0,
+        # KeyIndex = 1, KeyID Mode = 2
+        _ed_pkts.filter_mle_cmd(
+            MLE_CHILD_ID_REQUEST).must_next().must_verify(lambda p: p.wpan.aux_sec.key_index == 1 and p.wpan.aux_sec.
+                                                          key_id_mode == 2 and p.wpan.aux_sec.key_source == 0)
+
+        # Step 3: Leader send an ICMPv6 Echo Request to DUT.
+        # The MAC Auxiliary security header must contain
+        # KeyIndex = 1, KeyID Mode = 1
+        lp = _leader_pkts.filter_ping_request().filter(
```

Comment:
should check the destination?

Reply:
There are only LEADER and ED in the test case, and will verify if ed sends ping reply for the ping request in next step

Reply:
added